src/core/sys/posix/iconv.d

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+
+    D binding for the POSIX iconv library.
+
+    See_Also:
+       http://pubs.opengroup.org/onlinepubs/009695399/functions/iconv_open.html
+
```

Comment:
I'm not sure, but I think every new file should have copyright and license, similar to e.g. https://github.com/dlang/druntime/blob/master/src/core/sys/posix/mqueue.d

@WalterBright is probably the right person to ask.